File: programmers/12906_2.py
# 같은 숫자는 싫어
# https://programmers.co.kr/learn/courses/30/lessons/12906
def solution(arr):
    answer = []
    for i in range(len(arr) - 1):
        if i == 0:
            answer.append(arr[i])
        if arr[i] != arr[i + 1]:
            answer.append(arr[i + 1])
    return answer

File: programmers/12906_2.cpp
// 같은 숫자는 싫어
// https://programmers.co.kr/learn/courses/30/lessons/12906
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) 
{
	vector <int> answer;
	arr.erase(unique(arr.begin(), arr.end()), arr.end());
	answer = arr;
    return answer;
}

/*
	참고
	https://dpdpwl.tistory.com/39
*/